vc/consent-credential-v1.jsonld

```diff
@@ -9,11 +9,37 @@
       "ConsentCredential":"ex:ConsentCredential",
       "issuerDIDDoc":{
          "@id":"ex:issuerDIDDoc",
-         "@type":"@json"
+         "@context":{
+            "@version":1.1,
+            "@protected":true,
+            "id":"@id",
+            "type":"@type",
+            "did":{
+               "@id":"ex:did",
+               "@type":"xsd:string"
+            },
+            "doc":{
+               "@id":"ex:doc",
+               "@type":"@json"
+            }
+         }
       },
       "rpDIDDoc":{
          "@id":"ex:rpDIDDoc",
```

Comment:
Left rpDIDDoc as it is, since we have userDID which is just a string.

Reply:
@rolsonquadras I don't understand

Reply:
1. 
```
{
   "userDID":"did:example:abc789",
   "rpDIDDoc":{
      "doc":{

      },
      "id":"did:peer:123456789abcdefghi#inbox"
   },
}
```

2.
```
{
   "userDID":"did:example:abc789",
   "rpDID":{
      "doc":{

      },
      "id":"did:peer:123456789abcdefghi#inbox"
   },
}
```
@llorllale what I meant was that pt1 makes more sense than pt2.